src/index.njk

```diff
@@ -78,8 +78,8 @@ description: Design your service using GOV.UK styles, components and patterns
       <div class="govuk-grid-column-two-thirds govuk-!-mb-r4">
         <h2 class="govuk-heading-l">Roadmap</h2>
         <p class="govuk-body">
-          See what the Design System team at GDS is planning to work on over the coming year in the
-          <a href="/roadmap" class="govuk-link" data-hcontribute="guidelinegh">GOV.UK Design System Roadmap</a>.
+          See what the Design System team at GDS is planning to work on over the next 12 months in the
+          <a href="https://github.com/orgs/alphagov/projects/5" class="govuk-link" data-hcontribute="guidelinegh">GOV.UK Design System Roadmap (on GitHub)</a>.
```

Comment:
Thoughts on linking to the fullscreen view? `https://github.com/orgs/alphagov/projects/5?fullscreen=true`

Reply:
Good call